permissions/schemas/schema.json

```diff
@@ -330,6 +330,18 @@
           "const": "deny-get",
           "markdownDescription": "Denies the get command without any pre-configured scope."
         },
+        {
+          "description": "Enables the is_native command without any pre-configured scope.",
+          "type": "string",
+          "const": "allow-is-native",
```

Comment:
Minor nit: Why is `is_native` snake case and `allow-is-native` kebab, but `registerListener` camelCase and `allow-registerListener` kebab-camelCase? Is there a way for it all to be consistent?

Reply:
Good question. I've observed some differences in how naming/casing is handled in Tauri metadata i.e. permissions, Rust and native plugin code.

The notable difference is that `registerListener` is an in-built command which is implemented in Tauri's Swift and Kotlin API. Whereas `is_native` is a custom command which is implemented in Rust only. I found if I make isNative camel-cased for the life of me I cannot get it to bind to the command. This could be a Tauri bug. If you're happy, I'd like to follow this up a point for discovery outside of this PR.

Reply:
Sure, that'd be fine. Thanks for the explanation.